packages/neos-ui-editors/src/Editors/Range/index.js

```diff
@@ -68,11 +68,11 @@ class RangeEditor extends PureComponent {
     };
 
     render() {
-        const {options} = this.props;
+        const options = Object.assign({}, this.constructor.defaultProps.options, this.props.options);
```

Comment:
```suggestion
        const {options} = this.props;
```

Reply:
i looks like the 'static defaultProps' is a react thing and works out of box: https://stackoverflow.com/questions/38786973/how-to-set-component-default-props-on-react-component

so maybe your good to go using only this.props.options and the rest is handeled backstage...

Reply:
Nope, this didn't work out, but maybe @markusguenther has a better opinion as my approach

Reply:
No that would not work. But we can use the spread operator here.
```const options = {...this.constructor.defaultProps.options, ...this.props.options};```